examples/protonect/src/registration.cpp

```diff
@@ -72,60 +72,62 @@ void Registration::depth_to_color(float mx, float my, float& rx, float& ry)
     (mx * mx * color.my_x2y0) + (my * my * color.my_x0y2) + (mx * my * color.my_x1y1) +
     (mx * color.my_x1y0) + (my * color.my_x0y1) + (color.my_x0y0);
 
-  rx = wx / (color.fx * color_q);
-  ry = wy / (color.fx * color_q);
+  rx = (wx / (color.fx * color_q)) - (color.shift_m / color.shift_d);
+  ry = (wy / (color.fx * color_q)) * color.fy + color.cy;
```

Comment:
I think it should be `fy`, I didn't checked the previous code for errors. But from the senser provided intrinsics `fy` and `fx` are always the same. But if someone uses other intrinsics this could lead to wrong values, I will change it.